consul/datadog_checks/consul/consul.py

```diff
@@ -217,7 +217,15 @@ def get_nodes_with_service(self, instance, service):
 
         return self.consul_request(instance, consul_request_url)
 
-    def _cull_services_list(self, services, service_whitelist, max_services=MAX_SERVICES):
+    def _cull_services_list(self, services, service_whitelist, service_blacklist=None, max_services=MAX_SERVICES):
+
+        if not service_blacklist:
+            service_blacklist = []
+
+        print(services, service_blacklist)
+        services = {
+            s: services[s] for s in services if s not in service_blacklist
+        }
```

Comment:
For blacklisting, can you use this function that was just added in the base check? https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/utils/common.py#L14 We're trying to consolidate the way we do this across the whole codebase.

Reply:
I can, but I think either there's a bug in that function or it doesn't do quite what we would need here:

https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/utils/common.py#L47

I think that in the case that there's a blacklist, then the `items` have the blacklisted items removed but they're not restricted to _just_ the `whitelisted` items.